ui/src/importer/importerTaskDetails/ImporterTaskDetailsComponents/ImportedTable.js

```diff
@@ -35,7 +35,7 @@ export class ImportedTable extends React.Component {
                 Action
               </Table.HeaderCell>
               <Table.HeaderCell collapsing rowSpan="3">
-                Output document
+                Related document
```

Comment:
Comment: If this change was discussed with the library then it's good to me. Otherwise be aware that this will affect to correclty processed records as well.
I would check with them before.

Reply:
This was discussed! 😄